tests/test_umbral.py

```diff
@@ -13,7 +13,7 @@
     ]
 
 
-def test_encrypt_decrypt():
+def test_encapsulation():
```

Comment:
This isn't quite accurate. Encapsulation is a cryptographic process as a means to store encrypted keys in a specified format. I think the prior name is clear on this.

Reply:
Wait, what?

"encrypt / decrypt" is completely generic.  It says nothing about what the test is testing.

And this test does in fact test encapsulation, showing that the decapsulated key is `==` to the encapsulated one.  

Perhaps it's not an ideal name, but surely it's not less accurate than `test_encrypt_decrypt`?

Reply:
Well, now that you mention, the intention of this test is to test decapsulation of original capsules. Perhaps `test_decapsulation_original_capsule`?

Reply:
Yep, that's perfect.  Also really good, if we're going to use the character names in these tests, is `test_decapsulation_by_alice`